sdk/nodejs/src/client.ts

```diff
@@ -73,6 +77,9 @@ export class DevCycleClient {
                 this.configHelper = new EnvironmentConfigManager(
                     this.logger,
                     sdkKey,
+                    setConfigDataUTF8,
+                    setInterval,
+                    clearInterval,
```

Comment:
The interface for these is different between NodeJS / Web, we need to pass it in as an interface.